src/string_object.cpp

```diff
@@ -657,6 +657,14 @@ Value StringObject::split(Env *env, Value splitter, Value max_count_value) {
     }
 }
 
+bool StringObject::include(Env *env, Value arg) {
+    auto to_str = "to_str"_s;
+    if (!arg->is_string() && arg->respond_to(env, to_str))
+        arg = arg->send(env, to_str);
+    arg->assert_type(env, Object::Type::String, "String");
+    return ::strstr(c_str(), arg->as_string()->c_str()) != nullptr;
```

Comment:
I don't think we can use `strstr` (or even `strnstr`) here because it's possible to generate a Ruby string containing a null character in the middle:

```ruby
"foo\x00bar".include?("bar") # => true
````